platypus/platypus.lua

```diff
@@ -143,29 +140,36 @@ function M.create(config)
 	local RAY_CAST_DOWN_ID = 6
 	local RAY_CAST_DOWN_LEFT_ID = 7
 	local RAY_CAST_DOWN_RIGHT_ID = 8
-
-	local RAY_CAST_LEFT = BOUNDS_LEFT + vmath.vector3(-1, 0, 0)
-	local RAY_CAST_RIGHT = BOUNDS_RIGHT + vmath.vector3(1, 0, 0)
-	local RAY_CAST_DOWN = BOUNDS_BOTTOM + vmath.vector3(0, -1, 0)
-	local RAY_CAST_UP = BOUNDS_TOP + vmath.vector3(0, 1, 0)
-	local RAY_CAST_DOWN_LEFT = RAY_CAST_LEFT + RAY_CAST_DOWN
-	local RAY_CAST_DOWN_RIGHT = RAY_CAST_RIGHT + RAY_CAST_DOWN
-	local RAY_CAST_UP_LEFT = RAY_CAST_UP + RAY_CAST_LEFT
-	local RAY_CAST_UP_RIGHT = RAY_CAST_UP + RAY_CAST_RIGHT
-
-	-- order of ray casts is important!
-	-- we need to check for wall contact before checking ground
-	-- contact to be able to handle collision separation properly
-	local RAYS = {
-		{ id = RAY_CAST_LEFT_ID, ray = RAY_CAST_LEFT },
-		{ id = RAY_CAST_RIGHT_ID, ray = RAY_CAST_RIGHT },
-		{ id = RAY_CAST_UP_ID, ray = RAY_CAST_UP },
-		{ id = RAY_CAST_UP_LEFT_ID, ray = RAY_CAST_UP_LEFT },
-		{ id = RAY_CAST_UP_RIGHT_ID, ray = RAY_CAST_UP_RIGHT },
-		{ id = RAY_CAST_DOWN_ID, ray = RAY_CAST_DOWN },
-		{ id = RAY_CAST_DOWN_LEFT_ID, ray = RAY_CAST_DOWN_LEFT },
-		{ id = RAY_CAST_DOWN_RIGHT_ID, ray = RAY_CAST_DOWN_RIGHT },
-	}
+ 
```

Comment:
The indentation is messed up. Please use the same indentation (in this case tabs)

Reply:
yes, i didn't noticed this (my indentation use space) when modifying. It's fixed.